ee/clickhouse/models/property.py

```diff
@@ -364,6 +374,11 @@ def get_property_string_expr(
         (optional) alias of the table being queried
     :return:
     """
+
+    for reserved_words in EVENT_ATTRIBUTE_RESERVED_PROPERTIES_BY_TYPE.values():
+        if property_name in reserved_words:
+            return property_name, False
```

Comment:
This doesn't work due to table aliases and will cause queries to fail due to clashing column names.